library/sequencer/src/lib/ui/sequence.rb

```diff
@@ -0,0 +1,77 @@
+require "yast"
+Yast.import "Sequencer"
+
+module UI
+  # A {UI::Sequence} is an object-oriented interface for the good old
+  # {Yast::SequencerClass Yast::Sequencer}.
+  # In the simple case it runs a sequence of dialogs
+  # connected by Back and Next buttons.
```

Comment:
as sequencer can be a bit tricky, I really welcome here one or two trivial examples of usage